service/dependencies.gradle

```diff
@@ -1,31 +1,18 @@
-// Transitive dependencies pulled in via Spring dependency management need to be
-// modified via dependency-management plugin settings. Otherwise, the plugin
-// will just clobber versions. See the list of modifiable versions here:
-// https://docs.spring.io/spring-boot/docs/current/reference/html/dependency-versions.html#appendix.dependency-versions.properties
-project.ext {
-  set("snakeyaml.version", "1.33")
-  // opentelemetry-bom must have version >= 1.34.1 for compatibility with terra-common-lib 1.0.9:
-  set("opentelemetry.version", "1.36.0")
-}
 // Library Dependencies, Versions, etc.
+// In order for Dependabot to update these dependencies, this file must live at the root of its project.
 dependencies {
-  // Google dependencies
-  constraints {
-    implementation 'com.google.guava:guava:31.1-jre' // "-jre" for Java 8 or higher
-  }
+  // Google dependencies versioned by bom
   implementation platform('com.google.cloud:libraries-bom:26.34.0')
-
   implementation 'com.google.cloud:google-cloud-bigquery'
   implementation 'com.google.cloud:google-cloud-bigquerydatatransfer'
-  implementation group: "com.google.auto.value", name: "auto-value-annotations"
-  implementation group: "com.google.guava", name:"guava"
+  implementation 'com.google.auto.value:auto-value-annotations'
 
   // Azure dependencies
   implementation 'com.azure:azure-storage-blob:12.21.1'
```

Comment:
I wanted to upgrade this to latest version, 12.25.3, but [rolled back](https://github.com/DataBiosphere/terra-workspace-manager/pull/1686/commits/a8ae3cef39f66c9a7c1eb0e6a1fbe39a08fe1af4) due to a failing unit test.

https://scans.gradle.com/s/opkisam4oa35i/tests/task/:service:azureUnitTest/details/bio.terra.workspace.service.resource.controlled.cloud.azure.AzureStorageAccessServiceUnitTest/createAzureStorageContainerSasToken_computesAuthHash()?top-execution=1

Flagging for @aherbst-broad who wanted to take a closer look.

Reply:
From Drew in Slack:

>I think the sas token hash breakage is expected, the service version has changed from
>sv=2021-12-02
>to
>sv=2023-11-03

So, I'll add it back in and update the auth hash in our test.